lib/config/validation.ts

```diff
@@ -884,12 +882,6 @@ async function validateGlobalConfig(
   currentPath: string | undefined,
   config: RenovateConfig,
 ): Promise<void> {
-  if (getDeprecationMessage(key)) {
-    warnings.push({
-      topic: 'Deprecation Warning',
-      message: getDeprecationMessage(key)!,
-    });
-  }
```

Comment:
why this? can we use istanbul ignore here for future use?

Reply:
Good point